site/ko/alpha/guide/using_gpu.ipynb

```diff
@@ -0,0 +1,588 @@
+    "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+    "\n",
+    "!pip install tf-nightly-gpu-2.0-preview\n",
+    "import tensorflow as tf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "UhNtHfuxCGVy"
+   },
+   "source": [
+    "## 장치 할당 로깅\n",
+    "\n",
+    "연산과 텐서가 어떤 장치에 할당되었는지 확인하려면 `tf.debugging.set_log_device_placement(True)`를 프로그램의 가장 처음에 선언하세요. 장치 할당 로깅을 활성화하면 모든 텐서나 연산 할당이 출력됩니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {
+     "height": 85
+    },
+    "colab_type": "code",
+    "id": "2Dbw0tpEirCd",
+    "outputId": "2ae5cf66-2cdf-49b2-d8d0-87492e97e137"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Executing op MatMul in device /job:localhost/replica:0/task:0/device:GPU:0\n",
+      "tf.Tensor(\n",
+      "[[ 22.  28.]\n",
+      " [ 49.  64.]], shape=(2, 2), dtype=float32)\n"
+     ]
+    }
+   ],
+   "source": [
+    "tf.debugging.set_log_device_placement(True)\n",
+    "\n",
+    "# 텐서 생성\n",
+    "a = tf.constant([[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]])\n",
+    "b = tf.constant([[1.0, 2.0], [3.0, 4.0], [5.0, 6.0]])\n",
+    "c = tf.matmul(a, b)\n",
+    "\n",
+    "print(c)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "kKhmFeraTdEI"
+   },
+   "source": [
+    "위 코드는 `MatMul` 연산이 `GPU:0`에서 수행되었다고 보여줄 것입니다."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "U88FspwGjB7W"
+   },
+   "source": [
+    "## 장치 수동 할당\n",
+    "\n",
+    "특정 연산을 수행할 장치를 직접 선택하고 싶다면, `with tf.device`로 장치 컨텍스트를 생성할 수 있고 해당 컨텍스트에서의 모든 연산은 지정된 장치에서 수행됩니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {
+     "height": 85
+    },
+    "colab_type": "code",
+    "id": "8wqaQfEhjHit",
+    "outputId": "f96e5676-0dbc-462d-fde9-7c9831bfd01a"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Executing op MatMul in device /job:localhost/replica:0/task:0/device:GPU:0\n",
+      "tf.Tensor(\n",
+      "[[ 22.  28.]\n",
+      " [ 49.  64.]], shape=(2, 2), dtype=float32)\n"
+     ]
+    }
+   ],
+   "source": [
+    "tf.debugging.set_log_device_placement(True)\n",
+    "\n",
+    "# 텐서를 CPU에 할당\n",
+    "with tf.device('/CPU:0'):\n",
+    "  a = tf.constant([[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]])\n",
+    "  b = tf.constant([[1.0, 2.0], [3.0, 4.0], [5.0, 6.0]])\n",
+    "\n",
+    "c = tf.matmul(a, b)\n",
+    "print(c)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "8ixO89gRjJUu"
+   },
+   "source": [
+    "`a`와 `b`가 `CPU:0`에 할당되었습니다. `MatMul` 연산은 수행할 장치가 명시적으로 할당되어 있지 않기 때문에 텐서플로 런타임(runtime)은 연산과 가용한 장치들(이 예제에서는 `GPU:0`)을 기반으로 하나를 고를 것이고 필요하다면 장치들간에 텐서를 자동적으로 복사할 것입니다."
```

Comment:
Most great writers recommend not to use '~적'.
It was often used to translate Japanese as it is.
Unfortunately it can't avoid completely.
But you can write better, if you don't use '~적' as much as possible.
I suggest '자동적으로' --> '자동으로' :)

Reply:
Thank you for the tip! 👍